confgenerator/config.go

```diff
@@ -356,23 +357,31 @@ type LoggingPipeline struct {
 // Ops Agent metrics config.
 type metricsReceiverMap map[string]MetricsReceiver
 type metricsProcessorMap map[string]MetricsProcessor
-type metricsExporterMap map[string]MetricsExporter
 type Metrics struct {
 	Receivers  metricsReceiverMap  `yaml:"receivers" validate:"dive,keys,startsnotwith=lib:"`
 	Processors metricsProcessorMap `yaml:"processors" validate:"dive,keys,startsnotwith=lib:"`
 	// Exporters are deprecated and ignored, so do not have any validation.
-	Exporters metricsExporterMap `yaml:"exporters,omitempty"`
-	Service   *MetricsService    `yaml:"service"`
+	Exporters map[string]interface{} `yaml:"exporters,omitempty"`
```

Comment:
We will now allow arbitrary YAML in the `exporters` field, and it's completely unvalidated. Is this where we want to be with respect to customer configs? Or do we want to restrict what they put in there before we ignore it?

Reply:
We already dropped validation on this field, both partially before and now even more. It's a lot of work to maintain validation on a field that's already ignored and deprecated. Anyone who is specifying this field has been doing it since pre-2.0, so they already had a chance to get the field validated. I don't think it's worth the effort to maintain unused validation.